src/client/Client.js

```diff
@@ -402,10 +408,26 @@ class Client extends BaseClient {
 
     const query = new URLSearchParams({
       client_id: this.application.id,
-      scope: 'bot',
     });
 
-    if (options.permissions) {
+    const scopes = options.scopes;
+    if (typeof scopes === 'undefined') {
+      throw new TypeError('INVITE_MISSING_SCOPES');
+    }
+    if (!Array.isArray(scopes)) {
+      throw new TypeError('INVALID_TYPE', 'scopes', 'Array of Invite Scopes', true);
+    }
+    const requiredScope = scopes.find(scope => scope === 'bot' || scope === 'applications.commands');
+    if (!requiredScope) {
+      throw new TypeError('INVITE_MISSING_SCOPES');
+    }
+    const invalidScope = scopes.find(scope => !InviteScopes.includes(scope));
+    if (invalidScope) {
```

Comment:
```suggestion
    if (scopes.some(scope => !InviteScopes.includes(scope))) {
```

Reply:
Can't make this change - the TypeError returns `invalidScope` to the user